lib/publiccloud/ssh_interactive.pm

```diff
@@ -22,13 +22,14 @@ our @EXPORT = qw(ssh_interactive_tunnel ssh_interactive_leave select_host_consol
 sub establish_tunnel_console {
     my ($console) = @_;
 
-    select_console("$console");
+    select_console("$console", timeout => 180);
     # Note: Don't use script_run here! The serial terminal is set to /dev/sshserial, so every script_run will time out
     type_string("\n~.\n", max_interval => 1);    # ensure no previous ssh connection is present
     enter_cmd("clear");
-    enter_cmd('ssh -t sut');
-    # give the ssh connection some time to settle
-    sleep 5;
+    # Note: add one more "\n" in case of the original "\n" was missed sporadically
+    enter_cmd("ssh -t sut\n", max_interval => 1);
+    # Double check the console
+    select_console("$console", timeout => 180);
```

Comment:
I think this is not true.

Reply:
You are right.
I removed the `sleep 5;` and added the `Double check the console` code.
I just think it is better. Should I remove the code changes in lib/publiccloud/ssh_interactive.pm?

Reply:
No, the `select_console` is called from elsewhere. Please undo this whole change.

Reply:
Sure, I will remote the code changes in ssh_interactive.pm.